debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RowDeserializers.java

```diff
@@ -295,16 +296,15 @@ protected static Serializable deserializeTime(ByteArrayInputStream inputStream)
         int hours = split[2];
         int minutes = split[1];
         int seconds = split[0];
-        int nanoOfSecond = 0; // This version does not support fractional seconds
-        return LocalTime.of(hours, minutes, seconds, nanoOfSecond);
+        return Duration.ofHours(hours).plusMinutes(minutes).plusSeconds(seconds);
     }
 
     /**
-     * Converts a MySQL {@code TIME} value <em>with fractional seconds</em> to a {@link LocalTime}.
+     * Converts a MySQL {@code TIME} value <em>with fractional seconds</em> to a {@link java.time.Duration}.
      * 
      * @param meta the {@code meta} value containing the fractional second precision, or {@code fsp}
      * @param inputStream the binary stream containing the raw binlog event data for the value
-     * @return the {@link LocalTime} object
+     * @return the {@link java.time.Duration} object
      * @throws IOException if there is an error reading from the binlog event data
      */
     protected static Serializable deserializeTimeV2(int meta, ByteArrayInputStream inputStream) throws IOException {
```

Comment:
Having some unit tests for these methods would be great.

Reply:
I just added test for things that were already there. Would it make sense to create a separate ticket to add tests for all RowDeserializers? But I can still start a test suite for it, if you want to have it.

Reply:
Did you end up with adding a unit test for this method? As you say it'd be a good starting point (no need for adding test for all the other methods as part of this PR).